### ISSUE TITLE
Master branch's history has been altered

### ISSUE THREAD

**Issue by Dr-Emann** [AgentD/squashfs-tools-ng#92]
Just want to check if this is intentional. I've seen master be force updated a few times recently. It looks like new commits are coming in, and the top two commits are being rebased on top of new changes, as far as I've checked, the diff content of the changed commits are identical: the hashes are changing only because of different parents.

Here's the history I've caught at least: Note that master used to point at f6d0563, and before that was at 5781bc7

```
* 664099f - (HEAD, upstream/master, upstream/HEAD) Cleanup: table driven pax header parsing (11 hours ago)
* b71e8af - Cleanup: pax header parsing (11 hours ago)
* 72f9a50 - sqfs_dir_tree_destroy/sqfs_destroy: allow NULL input (11 hours ago)
* 1deb54c - (upstream/fixes-1.1.0) Add a test case for the path traversal bug (12 hours ago)
| * f6d0563 - Cleanup: table driven pax header parsing (2 days ago)
| * ca3e089 - Cleanup: pax header parsing (2 days ago)
| * 60db5e1 - sqfs_dir_tree_destroy/sqfs_destroy: allow NULL input (2 days ago)
|/
* ad109e2 - Fix symlink path traversal in rdsqaushfs (2 days ago)
* 4188e94 - Add an explicit license boiler plate to all the example programs (2 days ago)
* f9d242a - Add a simple example to show how to extract a single text file (2 days ago)
* 23b81d2 - Replace format.txt with an asciidoc version (2 days ago)
* 915808a - Strip space from first column of format.txt (2 days ago)
* 10bdc4e - Documentation: fix back reference to data block secton (2 days ago)
| * 5781bc7 - Cleanup: table driven pax header parsing (4 weeks ago)
| * 25d1702 - Cleanup: pax header parsing (4 weeks ago)
|/
* 2484949 - Fix libsquashfs directory writer size accounting (4 weeks ago)
```

**Comment by AgentD** [comment]
Yes, I know it is not very nice to this, but I intentionally rebased the `master` branch on top of `fixes-1.1.0` here.

I specifically don't want b71e8af and 664099f in the upcoming patch level release (IMO this cleanup still needs more time and I don't want to risk breaking something here), while on the other hand I don't want to fork off the `fixes-1.1.0` branch just yet and having to cherry-pick back into `master` all the time.

**Comment by Dr-Emann** [comment]
Okay, was just making sure it was intentional, thanks for the explanation.
